content/projects/python-specific/file-io/_index.md

```diff
@@ -33,7 +33,7 @@ charlie.save() # results in visitor_charley_sheen.json
 
 Notice that the full name used in the file is all lower-case and spaces are replaced by underscores.
 
-3. Create a function called `load` that takes in a name and then grabs a Visitor object from file. It should return the visitors instance.
+3. Create a method called `load` that takes in a name and then grabs a Visitor object from file. It should return the visitors instance.
```

Comment:
If you are going with the definition: "method is associated with an object, while a function is not." then this might confuse some people.  Since this method actually doesn't have a `self` or `this`, it returns an object.

The class itself is an object in Python, but things work differently in other languages. I'm not sure if the terminology would always be the same here.